### DISCUSSION TITLE
Cron jobs not executing in Docker container

### DISCUSSION THREAD

**Discussion by dpezet** [wavelog/wavelog#815] [Q&A] [answered]
I have Wavelog running in a Docker container following the instructions in the Wiki. Everything has been working well, except the cron jobs do not appear to be running. The cron jobs are enabled and the master cron job shows as OK. Each job shows an upcoming execution time, but they do not run. What is the best way to troubleshoot this issue? Thanks. 

![image](https://github.com/user-attachments/assets/94ec3479-b786-43ec-9688-ed7372df7412)

**Comment by HB9HIL** [comment, marked as answer]
Hello @dpezet 

There are some bugfixes in the current dev branch. We will release a new version soon so please stay patient. I will inform you when you cann repull your container image :)

**Reply by dpezet** [reply]
Excellent. I'll keep an eye out for the update. Thank you!
